x-pack/plugins/alerting/server/application/rule/methods/bulk_edit/bulk_edit_rules.ts

```diff
@@ -489,7 +489,7 @@ async function updateRuleAttributesAndParamsInMemory<Params extends RuleParams>(
       context,
       operations,
       rule: ruleDomain,
-      ruleActions,
+      ruleActions: ruleActions as RuleDomain['actions'], // TODO (http-versioning) Remove this cast once we fix injectReferencesIntoActions
```

Comment:
I though that bulkEdit was part of the versioning!

Reply:
The method isn't versioned but the route IOs are all versioned. 

Also I can remove this cast